src/payment/strategies/klarnav2/klarnav2-payment-initialize-options.ts

```diff
@@ -0,0 +1,28 @@
+import { KlarnaLoadResponse } from './klarna-payments';
+
+/**
+ * A set of options that are required to initialize the Klarna payment method.
+ *
+ * When Klarna is initialized, a widget will be inserted into the DOM. The
+ * widget has a list of payment options for the customer to choose from.
+ */
+export default interface KlarnaV2PaymentInitializeOptions {
+    /**
+     * The ID of a container which the payment widget should insert into.
+     */
+    container: string;
+    /**
+     * The payment_method_category specifies which of Klarna’s customer offerings
+     * (Pay now, Pay later or Slice it) that is being shown.
+     */
+    payment_method_category: string;
```

Comment:
If names are fixed could you create an `enum` for such purpose?
 
```javascript
export enum PaymentMethodCategory {
    PayNow = 'Pay Now',
    PayLater = 'Pay Later',
    SliceIt = 'Slice It'
}

Reply:
I think there is no need to do that change